packages/ui-extensions/docs/surfaces/point-of-sale/mdxExamples/discount-example/tile.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+
+import {
+  Tile,
+  useApi,
+  reactExtension,
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const SmartGridTile = () => {
+  const api = useApi<'pos.home.tile.render'>();
+
+  const shouldEnable = (subtotal: string): boolean => {
+    return Number(subtotal) > 100;
+  };
+
+  // [START tile.enable]
+  const [enabled, setEnabled] = useState(
+    shouldEnable(api.cart.subscribable.initial.subtotal),
+  );
+  // [END tile.enable]
+
+  // [START tile.subscribe]
+  api.cart.subscribable.subscribe((cart) => {
+    setEnabled(shouldEnable(cart.subtotal));
+  });
+  // [END tile.subscribe]
```

Comment:
There's a helper hook called useCartSubscription or useCartSubscribable that does this for you and returns a cart value that reflects the current value.